ibm/service/secretsmanager/data_source_ibm_sm_configurations.go

```diff
@@ -132,6 +138,61 @@ func DataSourceIbmSmConfigurations() *schema.Resource {
 							Computed:    true,
 							Description: "The name of the intermediate certificate authority.",
 						},
+						"crypto_key": &schema.Schema{
+							Type:        schema.TypeList,
+							MaxItems:    1,
+							Optional:    true,
```

Comment:
From below we are only setting crypto_key so this should be only computed : true

Reply:
Same comments holds for below datasources

Reply:
@hkantare fixed